cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml

```diff
@@ -17,13 +17,11 @@
   -->
 
 <!--
-  Logback used by containers that run in YARN eg. Flows, MR, Explore
+  Logback used by containers that run in YARN, such as Explore, Flows, MapReduce, Spark
 -->
 <configuration>
 
-  <!--
-    Disabling some chatty loggers.
-  -->
+  <!-- Disabling some chatty loggers -->
```

Comment:
Actually, this does not disable them. It "down-tunes" them. Errors still come through.